docs/docker/intro.md

```diff
@@ -0,0 +1,33 @@
+# Docker
+
+Docker is a container based virtualization software that allow it's user to create a development environment in a easy way without installing a virtualization software i.e VirtualBox.
+
+This allow developer to create multiple test environment on the fly and to share with others without too much headaches.
+
+# Architecture
+
+Docker architecture is centralized on the daemon. The daemon is used by all of the component of Docker. 
```

Comment:
Docker architecture is centralized on the daemon, which is used by all of Docker's components.

Reply:
Done